docs/data-model.asciidoc

```diff
@@ -378,6 +427,47 @@ include::./data-streams.asciidoc[tag=metrics-data-streams]
 
 See <<apm-data-streams>> to learn more.
 
+[float]
+==== Aggregated metrics: limits and overflows
```

Comment:
When this is merged, we should also link https://www.elastic.co/guide/en/kibana/8.8/troubleshooting.html#troubleshooting-too-many-transactions to this section.

Longer term, we could think about how to combine the UI-centric and the Server-centric documentation with something more wholistic.